README.md

```diff
@@ -112,7 +112,7 @@ If you are upgrading HackMD from an older version, follow these steps:
 5. Modify the file named `.sequelizerc`, change the value of the variable `url` with your db connection string
    For example: `postgres://username:password@localhost:5432/hackmd`
 6. Run `node_modules/.bin/sequelize db:migrate`, this step will migrate your db to the latest schema
-7. Start your whole new server!
+7. Start your whole new server! `nodejs app.js`
```

Comment:
I would recommend to use `npm start` instead.

Also when you check the [Install section](https://github.com/joamatab/hackmd/blob/b3f2453c5b45e62daacf27950667a6df758231ce/README.md#instructions) we tell people to use what ever they prefer (node, forever, pm2). So people should be aware of how to start their server again :)